README.md

```diff
@@ -3,3 +3,7 @@
 Hello and welcome to the repo for the magnetic user interface!
 
 💬 If you have any questions or need help, please reach out in the Slack channel.
+
+## Local development
```

Comment:
We'll need to add more detail to this down the road 👍

Reply:
I agree. Let's open another ticket for that work later.